ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/ChainDownloader.java

```diff
@@ -98,11 +95,16 @@ public void start() {
     }
   }
 
+  @VisibleForTesting
+  public CompletableFuture<?> getCurrentTask() {
```

Comment:
Can we make this package-private?

Reply:
Sadly no because `FullSyncDownloader` is under `...sync.fullsync` package and `FastSyncDownloader` will be under `...sync.fullsync` so this lives under `...sync`.  I think it's with this bit of pain to be able to have separate packages for fast sync vs full sync specific bits.